### PR TITLE
Evaka fix daily note fixes

### PR DESCRIPTION
**Pull request by reynders**
#### Summary
- Fix daycare group layout to match UX spec (more space, correct table header alignment)
- Fix daycare daily note group note text to be black, not gray
- Make employee unit group view daily note add / create button more obvious
- Show group note on employee mobile attendance UI
- Add sleeping hours to employee web UI tooltip

### CODE REVIEW

**Comment by massproduction** [comment]
Should we also add the group notes to `employee-mobile-frontend/components/attendances/notes/DailyNote.tsx`?

**Comment by reynders** [comment]
> Should we also add the group notes to `employee-mobile-frontend/components/attendances/notes/DailyNote.tsx`?

Is this ok frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNote.tsx

File: frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNote.tsx [hunk 1]
```diff
@@ -65,6 +66,12 @@ export default React.memo(function DailyNote({ child }: Props) {
...
           )}
+          {groupNote && (
+            <FixedSpaceColumn spacing={'xxs'}>
+              <Label>{i18n.attendances.notes.labels.groupNotesHeader}</Label>
+              <span>{groupNote.note}</span>
+            </FixedSpaceColumn>
+          )}
         </FixedSpaceColumn>
       ) : (
         <div>{i18n.attendances.notes.noNotes}</div>
```
Comment (massproduction) [inline frontend/src/employee-mobile-frontend/components/attendances/notes/DailyNote.tsx:77]: If the child does not have an daily note the groupNote will not be shown. How about adding the groupNote in different spot?

```suggestion
</FixedSpaceColumn>
      ) : (
        <>
          {groupNote ? (
            <FixedSpaceColumn spacing={'xxs'}>
              <Label>{i18n.attendances.notes.labels.groupNotesHeader}</Label>
              <span>{groupNote.note}</span>
            </FixedSpaceColumn>
          ) : (
            <div>{i18n.attendances.notes.noNotes}</div>
          )}
        </>
```

**Comment by massproduction** [comment]
If the child has only a group note then the ui will not show the daily note icon in the `ChildListItem` (because the check is just `attendanceChild.dailyNote [...] && (`).

The same thing is with `AttendanceChildPage` where the check for icon colour is `active={child && child.dailyNote ? true : false}`. 

Could we add the check for a group note for both of the se cases?

**Comment by massproduction** [comment]
We are not showing the number of hours slept in the tooltip in the desktop version of daily note. Just how the child slept.

**Review by massproduction — CHANGES_REQUESTED** [review]

**Review by massproduction — APPROVED** [review]
🏅

**Pull request merged**